docs/mkdocs.yml

```diff
@@ -36,17 +38,17 @@ theme:
   palette:
     primary: custom
     accent: deep orange
-        
   features:
     - navigation.tabs
+    - navigation.tabs.sticky
```

Comment:
the `sticky` was added because on the landing page somethime the `menu` was not visible (white text on white background)